frontend/src/lib/components/PropertyIcon.tsx

```diff
@@ -34,6 +34,14 @@ const osIcons = {
     ['android']: <IconAndroidOS />,
     ['ios']: <IconAppleIOS />,
     ['other']: <IconGearFilled />,
+    ['chrome os']: <IconChrome />,
+    ['windows mobile']: <IconWindows />,
+    ['windows phone']: <IconWindows />,
+    ['xbox']: <IconWindows />,
+    ['playstation']: <IconGearFilled />,
+    ['nintendo']: <IconGearFilled />,
```

Comment:
I'd prefer us to use the same we're using for `console` below for playstation/nintendo: `<IconHeadset />`

Reply:
Personally, I actually prefer it, this was the default "other" option for OS so I decided it could be the case that "Console" as a Headset makes sense, but the actual brands maybe were going to far. 

I did write to @corywatilo to check how we should approach this, but I am totally up to changing it to the headset in the meantime 👍

Reply:
+1 to headset for now, and Cory being the solution to our problems long-term :)

Where did you write to him? Tagging him on a thread in #web-analytics is probably the best way to do it, this way we can all follow the thread

Reply:
I DM him for some directions on how we should proceed but that does sounds better. I will wrap up stuff and move it there.